### ISSUE TITLE
Runs endless on Android API 21

### ISSUE THREAD

**Issue by vitusortner** [Malinskiy/action-android#15]
Using this GitHub Action with the following setup runs without an end.

```yaml
- name: Run instrumented tests
  uses: malinskiy/action-android/emulator-run-cmd@release/0.0.5
  with:
    cmd: ./gradlew connectedStagingDebugAndroidTest
    api: 21
    tag: default
    abi: x86
```

And the log output is the shown in the following.
```
2020-01-22T08:59:09.0330650Z ##[group]Run malinskiy/action-android/emulator-run-cmd@release/0.0.5
2020-01-22T08:59:09.0330840Z with:
2020-01-22T08:59:09.0330940Z   cmd: ./gradlew connectedStagingDebugAndroidTest
2020-01-22T08:59:09.0331040Z   api: 21
2020-01-22T08:59:09.0331120Z   tag: default
2020-01-22T08:59:09.0331220Z   abi: x86
2020-01-22T08:59:09.0331310Z   disableAnimations: false
2020-01-22T08:59:09.0331410Z   cmdOptions: -no-snapshot-save -noaudio -no-boot-anim
2020-01-22T08:59:09.0331520Z   verbose: false
2020-01-22T08:59:09.0331610Z env:
2020-01-22T08:59:09.0332100Z   SIGNING_KEY_PASSWORD: ***
2020-01-22T08:59:09.0332250Z   SIGNING_STORE_PASSWORD: ***
2020-01-22T08:59:09.0332350Z   ANDROID_HOME: /Users/runner/android-sdk
2020-01-22T08:59:09.0332450Z   ANDROID_SDK_ROOT: /Users/runner/android-sdk
2020-01-22T08:59:09.0332540Z   ANDROID_SDK_HOME: /Users/runner/android-sdk/sdk_home
2020-01-22T08:59:09.0332760Z   PATH: :/Users/runner/android-sdk/bin:/Users/runner/android-sdk/tools:/usr/local/lib/ruby/gems/2.6.0/bin:/usr/local/opt/ruby/bin:/usr/local/opt/curl/bin:/usr/local/bin:/usr/local/sbin:/Users/runner/bin:/Users/runner/.yarn/bin:/usr/local/go/bin:/Users/runner/Library/Android/sdk/tools:/Users/runner/Library/Android/sdk/platform-tools:/Users/runner/Library/Android/sdk/ndk-bundle:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/usr/bin:/bin:/usr/sbin:/sbin:/Users/runner/.azcopy/tools/bin:/usr/local/lib/ruby/gems/2.6.0/bin:/usr/local/opt/ruby/bin:/usr/local/opt/curl/bin:/usr/local/bin:/usr/local/sbin:/Users/runner/bin:/Users/runner/.yarn/bin:/usr/local/go/bin:/Users/runner/Library/Android/sdk/tools:/Users/runner/Library/Android/sdk/platform-tools:/Users/runner/Library/Android/sdk/ndk-bundle:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/usr/bin:/bin:/usr/sbin:/sbin:/Users/runner/.azcopy/platform-tools/bin
2020-01-22T08:59:09.0333320Z ##[endgroup]
2020-01-22T08:59:09.2111440Z Starting emulator with API=21, TAG=default and ABI=x86...
2020-01-22T08:59:09.2135590Z ANDROID_HOME is /Users/runner/android-sdk
2020-01-22T08:59:09.2153110Z PATH is :/Users/runner/android-sdk/bin:/Users/runner/android-sdk/tools:/usr/local/lib/ruby/gems/2.6.0/bin:/usr/local/opt/ruby/bin:/usr/local/opt/curl/bin:/usr/local/bin:/usr/local/sbin:/Users/runner/bin:/Users/runner/.yarn/bin:/usr/local/go/bin:/Users/runner/Library/Android/sdk/tools:/Users/runner/Library/Android/sdk/platform-tools:/Users/runner/Library/Android/sdk/ndk-bundle:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/usr/bin:/bin:/usr/sbin:/sbin:/Users/runner/.azcopy/tools/bin:/usr/local/lib/ruby/gems/2.6.0/bin:/usr/local/opt/ruby/bin:/usr/local/opt/curl/bin:/usr/local/bin:/usr/local/sbin:/Users/runner/bin:/Users/runner/.yarn/bin:/usr/local/go/bin:/Users/runner/Library/Android/sdk/tools:/Users/runner/Library/Android/sdk/platform-tools:/Users/runner/Library/Android/sdk/ndk-bundle:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/usr/bin:/bin:/usr/sbin:/sbin:/Users/runner/.azcopy/platform-tools/bin
2020-01-22T08:59:09.2154500Z [command]bash -c \/Users/runner/android-sdk/tools/bin/sdkmanager emulator tools platform-tools 'system-images;android-21;default;x86' > /dev/null
2020-01-22T08:59:11.2462140Z Warning: File /Users/runner/android-sdk/sdk_home/.android/repositories.cfg could not be loaded.
2020-01-22T09:00:09.9227930Z [command]bash -c \/Users/runner/android-sdk/tools/bin/sdkmanager 'platforms;android-21' > /dev/null
2020-01-22T09:00:11.2314330Z Warning: File /Users/runner/android-sdk/sdk_home/.android/repositories.cfg could not be loaded.
2020-01-22T09:00:19.7250300Z [command]/Users/runner/android-sdk/emulator/emulator -accel-check
2020-01-22T09:00:31.0915730Z accel:
2020-01-22T09:00:31.0916290Z 0
2020-01-22T09:00:31.0916880Z HAXM version 7.5.1 (4) is installed and usable.
2020-01-22T09:00:31.0917070Z accel
2020-01-22T09:00:31.0949900Z [command]bash -c \echo -n no | /Users/runner/android-sdk/tools/bin/avdmanager create avd -n emulator --package "system-images;android-21;default;x86" --tag default
2020-01-22T09:00:31.9927500Z Loading local repository...                                                     
2020-01-22T09:00:32.0015210Z [=========                              ] 25% Loading local repository...       
2020-01-22T09:00:32.0015780Z [=========                              ] 25% Fetch remote repository...        
2020-01-22T09:00:32.0016200Z [=========                              ] 25% Fetch remote repository...        
2020-01-22T09:00:32.0016610Z [=========                              ] 25% Fetch remote repository...        
2020-01-22T09:00:32.1372380Z [=======================================] 100% Fetch remote repository...       
2020-01-22T09:00:32.1374030Z Auto-selecting single ABI x86
2020-01-22T09:00:58.1770970Z Do you wish to create a custom hardware profile? [no] starting adb server
2020-01-22T09:00:58.1774630Z [command]/Users/runner/android-sdk/platform-tools/adb start-server
2020-01-22T09:00:58.3221780Z * daemon not running; starting now at tcp:5037
2020-01-22T09:01:01.6490710Z * daemon started successfully
2020-01-22T09:01:01.6511490Z [command]bash -c \/Users/runner/android-sdk/emulator/emulator @emulator -no-snapshot-save -noaudio -no-boot-anim &
2020-01-22T09:01:11.7526760Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:11.7591650Z error: device 'emulator-5554' not found
2020-01-22T09:01:11.7608530Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:11.7609130Z Sleeping for 1s
2020-01-22T09:01:12.7631280Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:12.7701650Z error: device 'emulator-5554' not found
2020-01-22T09:01:12.7802730Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:12.7803280Z Sleeping for 1s
2020-01-22T09:01:13.7848700Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:13.7976770Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:13.7977430Z error: device 'emulator-5554' not found
2020-01-22T09:01:13.7978550Z Sleeping for 1s
2020-01-22T09:01:14.8113670Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:14.8173150Z error: device 'emulator-5554' not found
2020-01-22T09:01:14.8182850Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:14.8183100Z Sleeping for 1s
2020-01-22T09:01:15.9653750Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:15.9714730Z error: device 'emulator-5554' not found
2020-01-22T09:01:15.9723380Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:15.9723620Z Sleeping for 1s
2020-01-22T09:01:16.9806430Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:16.9866000Z error: device 'emulator-5554' not found
2020-01-22T09:01:16.9875620Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:16.9875810Z Sleeping for 1s
2020-01-22T09:01:18.0612290Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:18.0675940Z error: device 'emulator-5554' not found
2020-01-22T09:01:18.0683120Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:18.0683330Z Sleeping for 1s
2020-01-22T09:01:19.1755550Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:19.2159440Z error: device 'emulator-5554' not found
2020-01-22T09:01:19.2374170Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:19.2374370Z Sleeping for 1s
2020-01-22T09:01:20.2404140Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:20.2595490Z error: device 'emulator-5554' not found
2020-01-22T09:01:20.2619590Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:20.2619790Z Sleeping for 1s
2020-01-22T09:01:21.3711760Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:21.3818310Z error: device 'emulator-5554' not found
2020-01-22T09:01:21.3854210Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:21.3854500Z Sleeping for 1s
2020-01-22T09:01:22.4665690Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:22.4748080Z error: device 'emulator-5554' not found
2020-01-22T09:01:22.4758340Z Sleeping for 1s
2020-01-22T09:01:22.4759270Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:23.4868970Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:23.4991620Z error: device 'emulator-5554' not found
2020-01-22T09:01:23.5070240Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:23.5070500Z Sleeping for 1s
2020-01-22T09:01:24.5201950Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:24.5238170Z error: device 'emulator-5554' not found
2020-01-22T09:01:24.5371970Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:24.5372350Z Sleeping for 1s
2020-01-22T09:01:25.6396690Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:25.6464060Z error: device 'emulator-5554' not found
2020-01-22T09:01:25.6572670Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:25.6572990Z Sleeping for 1s
2020-01-22T09:01:26.6578750Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:26.6643690Z error: device 'emulator-5554' not found
2020-01-22T09:01:26.6675560Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:26.6675850Z Sleeping for 1s
2020-01-22T09:01:27.6689330Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:27.6758260Z error: device 'emulator-5554' not found
2020-01-22T09:01:27.6815010Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:27.6815180Z Sleeping for 1s
2020-01-22T09:01:28.6826120Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:28.6893320Z error: device 'emulator-5554' not found
2020-01-22T09:01:28.6903290Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:28.6903470Z Sleeping for 1s
2020-01-22T09:01:29.8046760Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:29.8130430Z error: device 'emulator-5554' not found
2020-01-22T09:01:29.8167480Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:29.8167730Z Sleeping for 1s
2020-01-22T09:01:30.8205970Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:30.8283500Z error: device 'emulator-5554' not found
2020-01-22T09:01:30.8292540Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:30.8292780Z Sleeping for 1s
2020-01-22T09:01:31.8445810Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:31.8624150Z error: device 'emulator-5554' not found
2020-01-22T09:01:31.9097660Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:31.9098500Z Sleeping for 1s
2020-01-22T09:01:32.9977630Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:33.0042550Z error: device 'emulator-5554' not found
2020-01-22T09:01:33.0274800Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:33.0275090Z Sleeping for 1s
2020-01-22T09:01:34.0477020Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:34.0561020Z error: device 'emulator-5554' not found
2020-01-22T09:01:34.0740300Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:34.0740560Z Sleeping for 1s
2020-01-22T09:01:35.1076620Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:35.1154950Z error: device 'emulator-5554' not found
2020-01-22T09:01:35.1164670Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:35.1164910Z Sleeping for 1s
2020-01-22T09:01:36.1172000Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:36.1262150Z error: device 'emulator-5554' not found
2020-01-22T09:01:36.1267030Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:36.1269870Z Sleeping for 1s
2020-01-22T09:01:37.1949000Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:37.2090090Z error: device 'emulator-5554' not found
2020-01-22T09:01:37.2241530Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:37.2241860Z Sleeping for 1s
2020-01-22T09:01:38.3176970Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:38.3237920Z error: device 'emulator-5554' not found
2020-01-22T09:01:38.3250370Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:38.3250610Z Sleeping for 1s
2020-01-22T09:01:39.4633270Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:39.4691500Z error: device 'emulator-5554' not found
2020-01-22T09:01:39.4783660Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:39.4783960Z Sleeping for 1s
2020-01-22T09:01:40.5153280Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:40.5252290Z error: device 'emulator-5554' not found
2020-01-22T09:01:40.5260780Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:40.5260970Z Sleeping for 1s
2020-01-22T09:01:41.5372770Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:41.5432820Z error: device 'emulator-5554' not found
2020-01-22T09:01:41.5441780Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:41.5442000Z Sleeping for 1s
2020-01-22T09:01:42.5582020Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:42.5650030Z error: device 'emulator-5554' not found
2020-01-22T09:01:42.5663720Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:42.5663920Z Sleeping for 1s
2020-01-22T09:01:43.5707290Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:43.5767000Z error: device 'emulator-5554' not found
2020-01-22T09:01:43.5776010Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:43.5776470Z Sleeping for 1s
2020-01-22T09:01:44.6571360Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:44.6632670Z error: device 'emulator-5554' not found
2020-01-22T09:01:44.6642070Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:44.6642230Z Sleeping for 1s
2020-01-22T09:01:45.6657560Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:45.6720700Z error: device 'emulator-5554' not found
2020-01-22T09:01:45.6731380Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:45.6731530Z Sleeping for 1s
2020-01-22T09:01:46.6758070Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:46.6971040Z error: device 'emulator-5554' not found
2020-01-22T09:01:46.7015190Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:46.7015410Z Sleeping for 1s
2020-01-22T09:01:47.7149130Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:47.7210980Z error: device 'emulator-5554' not found
2020-01-22T09:01:47.7220640Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:47.7220820Z Sleeping for 1s
2020-01-22T09:01:48.7228120Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:48.7287560Z error: device 'emulator-5554' not found
2020-01-22T09:01:48.7360060Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:48.7360290Z Sleeping for 1s
2020-01-22T09:01:49.7414730Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:49.7473430Z error: device 'emulator-5554' not found
2020-01-22T09:01:49.7482670Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:49.7482840Z Sleeping for 1s
2020-01-22T09:01:50.7500620Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:50.7563050Z error: device 'emulator-5554' not found
2020-01-22T09:01:50.7627310Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:50.7627500Z Sleeping for 1s
2020-01-22T09:01:51.7657570Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:51.7720100Z error: device 'emulator-5554' not found
2020-01-22T09:01:51.7731060Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:51.7731230Z Sleeping for 1s
2020-01-22T09:01:52.6301100Z Unable to create basic Accelerated OpenGL renderer.
2020-01-22T09:01:52.6303330Z Unable to create basic Accelerated OpenGL renderer.
2020-01-22T09:01:52.6306440Z Core Image is now using the software OpenGL renderer. This will be slow.
2020-01-22T09:01:52.7847080Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:52.7909380Z error: device 'emulator-5554' not found
2020-01-22T09:01:52.7918420Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:52.7918600Z Sleeping for 1s
2020-01-22T09:01:53.7925920Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:53.8001780Z error: device 'emulator-5554' not found
2020-01-22T09:01:53.8008710Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:53.8008880Z Sleeping for 1s
2020-01-22T09:01:54.8266810Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:54.8441180Z error: device 'emulator-5554' not found
2020-01-22T09:01:54.8446230Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:54.8446980Z Sleeping for 1s
2020-01-22T09:01:55.4075100Z emulator: WARNING: encryption is off
2020-01-22T09:01:55.4075490Z emulator: WARNING: Running on a sytem with less than 8 logical cores. Setting number of virtual cores to 1
2020-01-22T09:01:55.4075910Z emulator: WARNING: Your GPU drivers may have a bug. Switching to software rendering.
2020-01-22T09:01:55.6837650Z emulator: ### WARNING: /etc/localtime does not point to zoneinfo-compatible timezone name
2020-01-22T09:01:55.6837940Z 
2020-01-22T09:01:55.8256050Z Your emulator is out of date, please update by launching Android Studio:
2020-01-22T09:01:55.8257790Z  - Start Android Studio
2020-01-22T09:01:55.8258650Z  - Select menu "Tools > Android > SDK Manager"
2020-01-22T09:01:55.8259140Z  - Click "SDK Tools" tab
2020-01-22T09:01:55.8259620Z  - Check "Android Emulator" checkbox
2020-01-22T09:01:55.8260040Z  - Click "OK"
2020-01-22T09:01:55.8260120Z 
2020-01-22T09:01:55.9727190Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:55.9808150Z error: device offline
2020-01-22T09:01:55.9877280Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:55.9877530Z Sleeping for 1s
2020-01-22T09:01:56.9884260Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:56.9955620Z error: device offline
2020-01-22T09:01:56.9962110Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:56.9962780Z Sleeping for 1s
2020-01-22T09:01:58.0037430Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:58.0129610Z error: device offline
2020-01-22T09:01:58.0137410Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:58.0137660Z Sleeping for 1s
2020-01-22T09:01:59.0147880Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:01:59.0335260Z error: device offline
2020-01-22T09:01:59.0377450Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:01:59.0377690Z Sleeping for 1s
2020-01-22T09:02:00.0413440Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:00.0549870Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:02:00.0550180Z error: device offline
2020-01-22T09:02:00.0551510Z Sleeping for 1s
2020-01-22T09:02:01.1917900Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:01.2052180Z error: device offline
2020-01-22T09:02:01.2095190Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:02:01.2095360Z Sleeping for 1s
2020-01-22T09:02:02.2101050Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:02.2161570Z error: device offline
2020-01-22T09:02:02.2170750Z The process '/Users/runner/android-sdk/platform-tools/adb' failed with exit code 1
2020-01-22T09:02:02.2170910Z Sleeping for 1s
2020-01-22T09:02:03.2570170Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:03.4761870Z 
2020-01-22T09:02:03.4782530Z Sleeping for 1s
2020-01-22T09:02:04.5255080Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:04.5400470Z 
2020-01-22T09:02:04.5421060Z Sleeping for 1s
2020-01-22T09:02:05.0170240Z emulator: ### WARNING: /etc/localtime does not point to zoneinfo-compatible timezone name
2020-01-22T09:02:05.0170980Z 
2020-01-22T09:02:05.6346820Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:05.6532230Z 
2020-01-22T09:02:05.6554950Z Sleeping for 1s
2020-01-22T09:02:06.7701300Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:06.7948400Z 
2020-01-22T09:02:06.7961850Z Sleeping for 1s
2020-01-22T09:02:07.8016710Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:07.8256640Z 
2020-01-22T09:02:07.8267440Z Sleeping for 1s
2020-01-22T09:02:08.8274920Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:08.8492180Z 
2020-01-22T09:02:08.8548700Z Sleeping for 1s
2020-01-22T09:02:09.8632260Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:09.8798400Z 
2020-01-22T09:02:09.8809880Z Sleeping for 1s
2020-01-22T09:02:10.9673400Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:10.9913510Z 
2020-01-22T09:02:10.9976180Z Sleeping for 1s
2020-01-22T09:02:12.1447010Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:12.1631740Z 
2020-01-22T09:02:12.1713960Z Sleeping for 1s
2020-01-22T09:02:13.2998300Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:13.3175400Z 
2020-01-22T09:02:13.3248820Z Sleeping for 1s
2020-01-22T09:02:14.4123190Z [command]/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 shell getprop sys.boot_completed
2020-01-22T09:02:14.4342600Z 
2020-01-22T09:02:14.4345770Z Sleeping for 1s
2020-01-22T09:02:15.5113660Z Timeout waiting for emulator to boot. Exiting
2020-01-22T09:02:15.5115590Z booted=false
2020-01-22T09:02:15.5119760Z ##[error]emulator boot failed
2020-01-22T09:02:58.0714540Z emulator: ### WARNING: /etc/localtime does not point to zoneinfo-compatible timezone name
2020-01-22T09:02:58.0721240Z 
2020-01-22T10:30:57.9518710Z ##[error]The operation was canceled.
```

**Comment by Malinskiy** [comment]
@vitusortner This issue should be fixed in 0.0.6 which includes https://github.com/Malinskiy/action-android/commit/9b53c25ce6ff2950230f4bb1445d47165d0c3bf1

**Comment by vitusortner** [comment]
I'm closing this as I didn't encounter the problem anymore 👍
